File: parte001/ex003_empleado_salario_neto.cs
// Un  empleado  trabaja  48  horas  en  la  semana  a  razón  de  $70.000  hora. 
// El  porcentaje  de  retención en la fuente es del 17.5% del salario bruto.
// Se desea saber cuál es el salario bruto, la retención en la fuente y el salario neto del trabajador.

using System;

namespace parte001
{
    class ex003_empleado_salario_neto
    {
        static void Main(string[] args)
        {
            double salarioBruto, salarioNeto, retencionFuente;

            salarioBruto = 48 * 70000;

            retencionFuente = salarioBruto * 0.175;

            salarioNeto = salarioBruto - retencionFuente;

            Console.WriteLine("El salario bruto es: " + salarioBruto);
            Console.WriteLine("La retención en la fuente es: " + retencionFuente);
            Console.WriteLine("El salario neto es: " + salarioNeto);
        }
    }
}

File: parte001/ex003_empleado_salario_neto.java
// Un  empleado  trabaja  48  horas  en  la  semana  a  razón  de  $70.000  hora. 
// El  porcentaje  de  retención en la fuente es del 17.5% del salario bruto.
// Se desea saber cuál es el salario bruto, la retención en la fuente y el salario neto del trabajador.

import java.util.Scanner;

public class ex003_empleado_salario_neto {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);

        System.out.print("Introduzca el salario por hora: ");
        double salarioHora = teclado.nextDouble();
        teclado.nextLine();

        System.out.print("Introduzca el número de horas trabajadas: ");
        double horasTrabajadas = teclado.nextDouble();
        teclado.nextLine();

        double salarioBruto = salarioHora * horasTrabajadas;
        double retención = 0.175 * salarioBruto;
        double salarioNeto = salarioBruto - retención;

        System.out.println("El salario bruto es: " + salarioBruto);
        System.out.println("La retención es: " + retención);
        System.out.println("El salario neto es: " + salarioNeto);

        teclado.close();
    }
}
